docs/releases/2.10.rst

```diff
@@ -104,3 +104,9 @@ Python 3.5 is no longer supported as of this release; please upgrade to Python 3
 
 In previous releases, rich text values were enclosed in a ``<div class="rich-text">`` element when rendered; this element has now been removed.
 To restore the old behaviour, see :ref:`legacy_richtext`.
+
+
+Hooks functions ``register_page_listing_buttons`` & ``register_page_listing_more_buttons``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Page listing button hook functions now accept an additional kwarg `next_url`.
```

Comment:
```suggestion
Page listing button hook functions now accept an additional kwarg ``next_url``. Please update your hook function definitions to accept the new ``next_url`` kwarg
```

Reply:
In applying this change I realise that we should document the purpose of `next_url`, since it's something that third-party code using these hooks might want to make use of. In turn, that makes me think that next_url should be the _unescaped_ version of the URL, since the escaping is just an implementation detail of how Wagtail's own functions pass that URL on to subsequent views.

Will create a follow-up PR to address this, so that it's not holding up this fix.